xilem_web/src/svg/kurbo_shape.rs

```diff
@@ -213,7 +213,7 @@ impl<State: 'static, Action: 'static> OrphanView<BezPath, State, Action, DynMess
             if attrs.was_created() {
                 attrs.push(("path", new.to_svg()));
```

Comment:
Doesn't this also need changed?

Reply:
Yeah, I think so. BTW, it was this way before #699 too.

Reply:
Or maybe it hasn't been. I am too sleepy to think coherently.